Characteristic-zh-cn.md

```diff
@@ -0,0 +1,38 @@
+[English (UK)](Colleges.md) | **简体中文（中国大陆）**
+
+# 家里蹲大学办学特色
+
+## 教学理念
```

Comment:
This shouldn't be like this

Following paragraphs are derived from University of Oxford. Maybe get some inspiration from them.

Reply:
would u give me a url about that?

Reply:
check ox.ac.uk